fcrepo-kernel-impl/src/test/java/org/fcrepo/integration/kernel/impl/FedoraResourceImplIT.java

```diff
@@ -647,6 +647,24 @@ public void testGetChildrenWithBinary() {
     }
 
     @Test
+    public void testGetChildrenWithBinaryCustomProperties() throws RepositoryException {
+        final String pid = getRandomPid();
+        final Container container = containerService.findOrCreate(session, "/" + pid);
+        final FedoraResource resource = binaryService.findOrCreate(session, "/" + pid + "/a");
+        final javax.jcr.Node node = resource.getNode();
+        node.setProperty("dc:title", "this-is-some-title");
+
+        final Graph graph =
+                container.getChildren().next().getTriples(subjects, PropertiesRdfContext.class).asModel().getGraph();
+
+        //jcr property
+        final Node s = createGraphSubjectNode(container.getChildren().next());
+        final Node p = DC_TITLE.asNode();
+        final Node o = createLiteral("this-is-some-title");
+        assertTrue(graph.contains(s, p, o));
```

Comment:
Leave an error message here for use in case of failure that explains why we _should_ see the triple included.